db_master_deploy/README.md

```diff
@@ -0,0 +1,80 @@
+database managment scripts
+==========================
+These scripts have been developped for the database and table duplication on the postgres master instance.
+The scripts have to be executed with ``geodata`` user. To become geodata you have to type:
+```bash
+$ sudo su - geodata
+```
+##Installation
+Clone the git repository to geodata's home:
+```bash
+$ git clone git@github.com:geoadmin/deploy.git
+$ cd deploy/db_master_deploy/
+```
+Make sure that geodata .pgpass file exists. It should contain the credentials for superuser ``pgkogis``
+```bash
+$ echo "*:5432:*:pgkogis:xxxxxxxx" >> /home/geodata/.pgpass
+$ chmod 0600 /home/geodata/.pgpass
+```
+
+The script output will be written to syslog ``/var/log/messages`` and can be analyzed with kibana interface.
+
+###database naming convention and service address
+The read-only hot-standby slaves can be accessed via this hostname: ``pg.bgdi.ch``
+Each database has to have one of the following suffixes
+
+DB Suffix    | Meaning
+-------------|------------|
+**_master** |  Original database, will be continuosly updated, should not be used in any applications |
+**_dev**   | development copy of _master database  | 
+**_int**   | integration copy of _master database | 
+**_prod**  | production copy of _master database  | 
+**_demo** |  demo copy of _master database |
+**_YYYYMMDD** | timestamped copy of _master database (bod only, use parameter -a YYYYMMDD for the archive suffix) |
```

Comment:
a good point, yes we could accept more than timestamps as suffix, the only restriction we have is that there should not be underscores in the suffix.
note: the ``timestamp/snapshot`` feature is currently only activated for the bod. other databases could be added, but too many snapshots of big database will make the diskspace grow too much.
```
^[a-zA-Z0-9]+$
```